build.gradle

```
@@ -68,5 +69,12 @@ dependencies {
 shadowJar {
     archiveFileName = 'denguetracker.jar'
 }
+run {
+    enableAssertions = true
+}
 
 defaultTasks 'clean', 'test'
+
+checkstyle {
+    toolVersion = '10.2'
+}
```

Should add EOF line.

```suggestion
}

```

---

done